data/modules/StationTrafficControl/StationTrafficControl.lua

```diff
@@ -106,8 +109,12 @@ end
 local loaded_data
 
 local onGameStart = function ()
+	local station = Game.player:FindNearestTo("SPACESTATION") 
```

Comment:
I assume this will never be nil, as can happen if the player is in hyperspace, but since we can't start the game in hyperspace, this should be fine, I assume.

Reply:
Fix pushed